### ISSUE TITLE
[123] Issue #123 - November 29, 2018

### ISSUE THREAD

**Issue by BasThomas** [SwiftWeekly/swiftweekly.github.io#424]
To contribute to this issue, simply leave a comment here. Please also review [our contributing guidelines](https://github.com/SwiftWeekly/swiftweekly.github.io/blob/master/CONTRIBUTING.md).

The current draft for this issue in [`_drafts/`](https://github.com/SwiftWeekly/swiftweekly.github.io/tree/master/_drafts). If you want to contribute directly, feel free to [open a pull request](https://github.com/SwiftWeekly/swiftweekly.github.io/compare?expand=1).

**Comment by solo-han** [comment]
https://forums.swift.org/t/draft-proposal-target-specific-build-settings/18031

**Comment by BasThomas** [comment]
https://twitter.com/jckarter/status/1063466207028768770

> When going out for a night of operator overloading with your friends, always bring a designated type. This proposal could address the infamous "expression too complex" problems with operators in Swift https://forums.swift.org/t/pitch-making-expression-type-checking-of-operator-expressions-fast/18037

**Comment by BasThomas** [comment]
https://github.com/apple/swift-evolution/blob/master/proposals/0237-contiguous-collection.md

**Comment by BasThomas** [comment]
234 in review?

**Comment by fassko** [comment]
https://forums.swift.org/t/lifting-the-self-or-associated-type-constraint-on-existentials/18025

Lifting the “Self or associated type” constraint on existentials

**Comment by fassko** [comment]
Cool article by @mattt about new Language Server Protocol (LSP) integrated in VS Code. I think this will open so many doors for Swift to be baked in many other editors. Making it easier to adapt in other platforms than macOS and other developments than iOS.

https://nshipster.com/vscode/

**Comment by fassko** [comment]
@balestrapatrick taking a deep dive into Swift Benchmark Suite and how to contribute to it.
https://patrickbalestra.com/blog/2018/11/12/contributing-to-the-swift-benchmark-suite.html

**Comment by BasThomas** [comment]
https://twitter.com/CodaFi_/status/1064969922974019584

> I wrote a little about contributing to Swift.  
>
> My DMs are open for just this kind of thing.  Please ask me questions.
>
> https://forums.swift.org/t/what-should-i-learn-if-i-want-to-contribute-to-the-swift-compiler/18144/4?u=codafi

**Comment by fassko** [comment]
https://jpsim.com/evaluating-swiftsyntax-for-use-in-swiftlint

tl;dr; Implementing SwiftLint using SwiftSyntax instead of SourceKitten would make it run over 20x slower 😭

https://twitter.com/harlanhaskins/status/1065775499526451200


<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by BasThomas** [comment]
https://twitter.com/BasThomas/status/1065499801490530304

> The video for "Taken For Granted", my presentation on the history of Swift @tryswiftnyc, is now online!
>
> Slides: https://speakerdeck.com/basthomas/taken-for-granted
https://www.youtube.com/watch?v=ePuOrCbIW-o https://twitter.com/BasThomas/status/1065499608145690626/photo/1

**Comment by fassko** [comment]
Can be a fun one 🙃

https://twitter.com/alexisgallagher/status/1066559309385883648

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by fassko** [comment]
https://twitter.com/swift_weekly/status/1067465597880082434

The #Swift team just landed String’s proposed final ABI on master. This ABI includes some significant changes, the primary one being that native Swift strings are stored as UTF-8.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by jpsim** [comment]
That was posted 22 days ago fyi

**Comment by fassko** [comment]
Tips and Tricks for Contributing to the Swift Language

https://pspdfkit.com/blog/2018/tips-for-contributing-to-the-swift-language/


<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
